app/models/drivetime_band.rb

```diff
@@ -2,4 +2,19 @@
 
 class DrivetimeBand < ApplicationRecord
   belongs_to :vha_facility, class_name: 'Facilities::VHAFacility'
+
+  class << self
+    def find_within_max_distance(lat, lng, drive_time, ids)
+      query = 'ST_Intersects(polygon, ST_MakePoint(:lng,:lat)) AND max <= :max'
+      params = { lng: lng, lat: lat, max: drive_time }
+
+      unless ids.nil?
```

Comment:
would it be better to check blank?

Reply:
the double negative throws me off a bit.
maybe `if ids.present?` ?